tests/test_create_language.rs

```diff
@@ -11,8 +11,6 @@ use postgres::Connection;
 
 use db::DatabaseHandler;
 
-/* TODO: #52 apply the Connection encapsulation to the whole file,
-   so we can then simply get rid of this direct module inclusion */
```

Comment:
so we still need the direct module iclusion ?

Reply:
I tried many combinations here. In the end, it seems that `connect_and_clean`, `assert_language_exists`... all require `use db::DatabaseHandler` that itself requires `mod db`.